__tests__/steps/navSteps.spec.ts

```diff
@@ -149,6 +160,20 @@ describe('navSteps', () => {
         load_fourth_screen_first_journey: true,
         load_third_screen_first_journey: true,
       });
+
+      // closing most recent step
+      markStep('loaded_third_screen_first_journey');
+
+      expect(getActiveStepsFromNavigationSteps()).toEqual({
+        load_fourth_screen_first_journey: true,
+        load_third_screen_first_journey: true,
+      });
     });
+
+    it('departs endmark if theres no start mark', () => { 
```

Comment:
So you are saying
`getActiveStepsFromNavigationSteps departs endmark if theres no start mark`

I am a bit confused with this wording `departs`, what is departing?

Reply:
consider `drops the endmark...` or `does nothing with the end mark...`